eldritch/encounters.py

```diff
@@ -151,17 +151,119 @@ def Asylum1(char):
+def Asylum5(char):
+  check = events.Check(char, "will", -1)
+  lose = events.Nothing() #TODO: Oh dear...so many choices
+  skill = events.Draw(char, "skills", 1)
+  cond = events.Conditional(char, check, "successes", {0: lose, 2: skill})
+  return events.Sequence([check, cond], char)
+def Asylum6(char):
+  check = events.Check(char, "lore", -2)
+  gain = events.Gain(char, {"clues": 2})
+  loss = events.Loss(char, {"stamina": 1})
+  return events.PassFail(char, check, gain, loss)
+def Asylum7(char):
+  check = events.Check(char, "fight", -2)
+  stamina = events.Gain(char, {"stamina": 2})
+  rest = events.Sequence([stamina, events.LoseTurn(char)], char)
+  fight = events.PassFail(char, check, events.Nothing(), rest)
+  return events.BinaryChoice(char, "Do you resist?", "Yes", "No", fight, rest)
 
 def Bank1(char):
-    return events.Nothing() # TODO: implement location choice
+  return events.Nothing() # TODO: implement location choice
+def Bank2(char):
+  check = events.Check(char, "luck", -1)
+  spend = events.Loss(char, {"dollars": 2})
+  common = events.Draw(char, "common", 1)
+  unique = events.Draw(char, "unique", 1)
+  cond = events.Conditional(char, check, "successes", {0: common, 1: unique})
+  prereq = events.AttributePrerequisite(char, "dollars", 2, "at least")
+  nothing = events.Nothing()
+  choice = events.BinaryChoice(char, "Pay $2 for man's last possession?",
+                              "Pay $2",
+                              "Let man and his family go hungry", 
+                              events.Sequence([spend, check, cond], char), nothing)
+  return events.PassFail(char, prereq, choice, nothing)
+def Bank3(char):
+  prep = events.CombatChoice(char, "Choose weapons to fight the bank robbers")
+  check = events.Check(char, "combat", -1)
+  robbed = events.Loss(char, {"dollars": char.dollars})
+  nothing = events.Nothing()
+  cond = events.Conditional(char, check, "successes", {0: robbed, 1: nothing})
+  return events.Sequence([prep, check, cond], char)
+def Bank4(char):
+  check = events.Check(char, "luck", -2)
+  bless = events.Bless(char)
+  curse = events.Curse(char)
+  return events.PassFail(char, check, bless, curse)
+def Bank5(char):
+  check = events.Check(char, "speed", -1)
+  gain = events.Gain(char, {"dollars": 2})
+  return events.PassFail(char, check, gain, events.Nothing())
+def Bank6(char):
+  return events.Loss(char, {"sanity": 1})
+def Bank7(char):
+  return events.GainOrLoss(char, {"dollars": 5}, {"sanity": 1})
 
 def Square1(char):
   return events.Gain(char, {"stamina": 1})
+def Square2(char):
+  check = events.Check(char, "will", -1)
+  # TODO: prerequisite ally being in the deck, otherwise two clue tokens
+  ally = events.DrawSpecific(char, "allies", "Fortune Teller")
+  return events.PassFail(char, check, ally, events.Nothing())
+def Square3(char):
+  check = events.Check(char, "will", -1)
+  loss = events.Loss(char, {'sanity': 1, 'stamina': 1})
```

Comment:
To keep things consistent, use double quotes here instead of single quotes.

Reply:
at spaceX almost everything tends to be single quotes, so I'm having to train myself